src/sentry/replays/usecases/summarize.py

```
@@ -135,69 +135,50 @@ def fetch_trace_connected_errors(
 
         # Process results and convert to EventDict objects
         error_events = []
-        seen_event_ids = set()  # Track seen event IDs to avoid duplicates
 
         # Process error query results
-        if error_query_results and "data" in error_query_results:
-            for event in error_query_results["data"]:
-                event_id = event.get("id")
-
-                # Skip if we've already seen this event
-                if event_id in seen_event_ids:
-                    continue
-
-                seen_event_ids.add(event_id)
-
-                timestamp = _parse_iso_timestamp_to_ms(
-                    event.get("timestamp_ms")
-                ) or _parse_iso_timestamp_to_ms(event.get("timestamp"))
-                message = event.get("message", "")
-
-                if timestamp:
-                    error_events.append(
-                        EventDict(
-                            category="error",
-                            id=event_id,
-                            title=event.get("title", ""),
-                            timestamp=timestamp,
-                            message=message,
-                        )
+        for event in error_query_results["data"]:
+            timestamp = _parse_iso_timestamp_to_ms(
+                event.get("timestamp_ms")
+            ) or _parse_iso_timestamp_to_ms(event.get("timestamp"))
+            message = event.get("message", "")
+
+            if timestamp:
+                error_events.append(
+                    EventDict(
+                        category="error",
+                        id=event.get("id"),
+                        title=event.get("title", ""),
+                        timestamp=timestamp,
+                        message=message,
                     )
+                )
 
         # Process issuePlatform query results
-        if issue_query_results and "data" in issue_query_results:
```

is there a chance the query results don't exist?

---

I think that's only when snuba errors, and we should raise rather than silently skip